app/components/Panel.js

```diff
@@ -38,57 +39,49 @@ export default class Panel extends Component {
       automaticallyRunQuery: undefined
     };
 
+    let backgroundPageConnection = chrome.runtime.connect({
+      // sending tabId as name to make connection one-step process
+      name: chrome.devtools.inspectedWindow.tabId.toString()
+    });
+
+    backgroundPageConnection.onMessage.addListener((request, sender) => {
+      request = [request];
```

Comment:
I tried to keep the code as similar to the old version. The comparable section of code in the old version returned the request through an eval in an array and mapped over it, so I ended up doing a similar thing.